--- openjdk/Inet4AddressImpl.c
+++ openjdk/Inet4AddressImpl.c
@@ -461,6 +461,21 @@
     return JNI_FALSE;
 }

+#ifdef WIN32
+DWORD WINAPI IcmpSendEcho2Ex(HANDLE,
+			     HANDLE,
+			     LPVOID,
+			     PVOID,
+			     IPAddr,
+			     IPAddr,
+			     LPVOID,
+			     WORD,
+			     LPVOID,
+			     LPVOID,
+			     DWORD,
+			     DWORD);
+#endif
+
 /**
  * ping implementation.
  * Send a ICMP_ECHO_REQUEST packet every second until either the timeout
