--- openjdk/java_props_md.c
+++ openjdk/java_props_md.c
@@ -212,17 +212,17 @@
          * SHELL32 DLL is delay load DLL and we can use the trick with
          * __try/__except block.
          */
-        __try {
+        /* __try { */
             /*
              * For Windows Vista and later (or patched MS OS) we need to use
              * [SHGetKnownFolderPath] call to avoid MAX_PATH length limitation.
              * Shell32.dll (version 6.0.6000 or later)
              */
             hr = SHGetKnownFolderPath(&FOLDERID_Profile, KF_FLAG_DONT_VERIFY, NULL, &u_path);
-        } __except(EXCEPTION_EXECUTE_HANDLER) {
+        /* } __except(EXCEPTION_EXECUTE_HANDLER) { */
             /* Exception: no [SHGetKnownFolderPath] entry */
-            hr = E_FAIL;
-        }
+        /*     hr = E_FAIL; */
+        /* } */
 
         if (FAILED(hr)) {
             WCHAR path[MAX_PATH+1];
